src/main/drivers/pwm_output.c

```diff
@@ -130,6 +130,7 @@ static pwmOutputPort_t *pwmOutConfig(const timerHardware_t *timerHardware, uint8
 
 static void pwmWriteBrushed(uint8_t index, uint16_t value)
 {
+    value = constrain(value, 1000, 2000);
```

Comment:
We should probably set the min and max allowable values for motors in the initialisation that way we can send varying resolutions (e.g. 2048 in the case of DSHOT)